applications/connectivity_bridge/src/modules/power_handler.c

```diff
@@ -56,7 +56,9 @@ static bool app_event_handler(const struct app_event_header *aeh)
 	}
 
 	if (is_power_down_event(aeh)) {
+#if defined(CONFIG_SOC_SERIES_NRF52X)
 		nrf_power_system_off(NRF_POWER);
```

Comment:
Wont the same work for 53 as well?

Reply:
NRF_POWER is not available in nRF53. Cortex-M33 goes to sleep in idle.